### PR TITLE
chore: fix extend docs fenced codeblocks in docs

### PR DESCRIPTION
**Pull request by jorenbroekema**
This was the only way for me to make it look good in storybook

### CODE REVIEW

File: docs/extending-documentation.md [hunk 1]
```diff
@@ -214,23 +214,34 @@ In the code above we have 2 places in where you can do overrides: `./.storybook/
...
 
-````md
-```
+Use triple backtick \` to create a fenced codeblock ` ``` `, and put inside it:
+
+```md
 ::addMdAfter(':root')
 ```
 
-```js script
```
Comment (daKmoR) [inline docs/extending-documentation.md:224]: did you try using 4 back ticks? or 3 `~` tildes as a guard around the guard?

Comment (jorenbroekema) [reply docs/extending-documentation.md:224]: I tried 4 back ticks, looks very ugly because the storybook md parser doesn't handle it well

Comment (jorenbroekema) [reply docs/extending-documentation.md:224]: I will try tildes now

Comment (jorenbroekema) [reply docs/extending-documentation.md:224]: Storybook's md parser will take ~~~ as opening and then the ``` as closing :( so also not possible

**Pull request merged**